operator/v1/0000_70_dns-operator_00.crd.yaml-patch

```diff
@@ -0,0 +1,5 @@
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/upstreamresolvers/properties/upstreams/items/properties/address/oneOf
+  value:
+    - format: ipv4
+    - format: ipv6
```

Comment:
oneOf is wrong. OneOf is really xor, i.e. if there happen to be addresses that are valid in ipv4 *and* ipv6, the validation would fail. It's likely not what you want. AnyOf is better (= or).